app/controlplane/internal/service/cascredential.go

```diff
@@ -103,7 +103,7 @@ func (s *CASCredentialsService) Get(ctx context.Context, req *pb.CASCredentialsS
 		}
 
 		// If we can't find a mapping, we'll use the default backend
-		if err != nil && !biz.IsNotFound(err) && !biz.IsErrUnauthorized(err) {
```

Comment:
I realized that this error was not useful in general since it was representing a NotFound, that's why I simplified it.